source/adaptive_load/scoring_function_impl.cc

```diff
@@ -0,0 +1,64 @@
+#include "adaptive_load/scoring_function_impl.h"
+
+#include <cmath>
+
+#include "envoy/registry/registry.h"
+
```

Comment:
nit: not sure if you meant for the extra spacing between headers.

Reply:
Yes, if you remove the lines, they come back after running ci/do_ci.sh fix_format. I think we are seeing are 5 1-item groups of includes, based on group prefixes in tools/check_format.sh.